### ISSUE TITLE
Allow rounding parameter at Decimal-based validators

### ISSUE THREAD

**Issue by the-infinity** [binary-butterfly/validataclass#88]
Sometimes it's necessary to define which type of rounding should be used at limiting output places. This could be an additional parameter called `rounding` which should be used as `Decimal.quantize()` `rounding` input.

**Comment by binaryDiv** [comment]
As I have just found out, the rounding mode in the default context of Python's `decimal` module is not *quite* what I (and most people?) would have expected: It uses the `ROUND_HALF_EVEN` method, which rounds a 5 either up OR down depending on the preceding number, rounding to even numbers. (For example, 1.5 would be rounded to 2, but 2.5 would be also rounded to 2, instead of 3.)

I would suggest to set the default value for the new `rounding` parameter to `decimal.ROUND_HALF_UP`. This would be a small breaking change, but I'd argue that for most applications this is actually more a bugfix than a change of expected behaviour.
To restore the default behaviour of using the decimal context, the parameter can simply be set to `None`.

So in a nutshell:

- New parameter `rounding: Optional[str] = decimal.ROUND_HALF_UP` (apparently these constants are strings)
- Without the `output_places` parameter, this parameter has no effect.
- Pass the rounding parameter to `decimal.quantize()`
- The documentation needs to be adjusted accordingly (right now it says the default decimal context is used)

**Comment by binaryDiv** [comment]
Implemented in #99.
